content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'This is the template for creating an article for the Arduino Documentation website.'
+tags: 
+  - article
+author: 'Lenard George Swamy'
+---
+
+## INTRODUCTION 
+
+Do you have an idea for your project but don’t know how to execute it for your class ? Use the web editor to start creating this project. For this example we chose to go with a simple project where you can read the distance sensor and make a proximity meter. 
```

Comment:
We should make this less "markety". The first sentence sounds like a beginning of a marketing video, instead of describing what this tutorial will go through.